app/api/url_generator.py

```diff
@@ -22,7 +26,10 @@ def get_api_url(user_query: str) -> str:
     Returns:
         str: The constructed API URL.
     """
-    base_api_url = "https://api.neurobagel.org/query/?"
+    base_api_url = os.getenv("NB_API_QUERY_URL")
+    if not base_api_url:
+        return "Error: The NB_API_QUERY_URL environment variable is not set. Please configure it and try again."
+
```

Comment:
I think it's better practice to raise a runtime error here similar to what we have in the api and check for that in the test:
https://github.com/neurobagel/api/blob/c7c1839d4a013c7a3d3b0567174dbf24e81c97f2/app/main.py#L92-L99